cwms-data-api/build.gradle

```diff
@@ -323,3 +324,8 @@ task generateTimeSeriesSamples(type: JavaExec) {
     args user
     args password
 }
+
+task executeOpenAPIConversion(type: Exec) {
+    workingDir rootDir
+    commandLine 'gradlew.bat', 'integrationTests', '--tests', 'cwms.cda.api.openapi.OpenApiYamlConversionIT'
```

Comment:
FYI, some of us use Linux.

Reply:
Also, this should just be a task of type Test that just limits the tests to the provided class.

you can do an `include "test name"` directly in the task, just like we exclude the integration test tag from the unit tests.